source/sass/buyers-guide/views/product.scss

```diff
@@ -1,7 +1,4 @@
-$product-detail-padding-x: (
```

Comment:
not needed as a variable since it's only used in one ruleset in this file (`.product-section-padding-x { }`)